.github/workflows/commit-queue.yml

```diff
@@ -17,9 +17,28 @@ env:
   NODE_VERSION: lts/*
 
 jobs:
-  commitQueue:
+  get_mergable_pull_requests:
```

Comment:
I pick this name because that's how it used to be called.

https://github.com/nodejs/node/blob/73fe15bf24d2166b2e5a37fc429b4eaef3537912/.github/workflows/commit-queue.yml#L51

I see the point for fix the typo though.

Reply:
It was a suggestion to change this and all others, for consistency with others workflows.